opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -154,6 +154,13 @@ impl LogProcessor for SimpleLogProcessor {
 /// them at a pre-configured interval.
 pub struct BatchLogProcessor<R: RuntimeChannel> {
     message_sender: R::Sender<BatchMessage>,
+
+    // Track dropped logs. We'll log this at shutdown and also emit
+    // as a metric.
+    dropped_logs_count: AtomicUsize,
+
+    // Track the maximum queue size that was configured for this processor
+    max_queue_size: usize,
```

Comment:
odd that we have to store this here just for logging purposes, but not an issue!

Reply:
I believe we can avoid this by moving the logging of dropped logs (otel_warn!) from the shutdown() method to the worker's Shutdown message processing.